generic_tracers/generic_tracer_utils.F90

```diff
@@ -1998,7 +2008,7 @@ subroutine g_tracer_get_string(g_tracer_list,name,member,string)
     character(len=*),         intent(in) :: name
     character(len=*),         intent(in) :: member
     type(g_tracer_type),    pointer    :: g_tracer_list, g_tracer 
-    character(len=fm_string_len), intent(out) :: string
```

Comment:
This is to avoid crash in debug mode. Apparently it's not right to specify the length on an intent(out) string!